docs/api-guide/routers.md

```diff
@@ -72,14 +72,22 @@ Alternatively you can use Django's `include` function, like so…
         url(r'^', include(router.urls)),
     ]
 
-Router URL patterns can also be namespaces.
+To include the router URL patterns with an application namespace pass a
+`(router.urls, app_name)` 2-tuple to `include`.
+
+For example:
 
     urlpatterns = [
         url(r'^forgot-password/$', ForgotPasswordFormView.as_view()),
-        url(r'^api/', include(router.urls, namespace='api')),
+        url(r'^api/', include((router.urls, 'app_name')),
```

Comment:
This is the >80% usage for namespaces. (We want to namespace our URL names to a single application.)

Reply:
The `include((` ... `)` looks wrong to me?

Reply:
I'm confused by the `app_name` argument, does it *have* to match the app name, or is it a general purpose namespace?

Reply:
The parenthesis is missing. I'll fix that. 

SO... 😃

What ultimately gets passed included is a 3-tuple: 

```
(A_LIST_OF_PATTERNS, AN_APPLICATION_NAMESPACE, AN_INSTANCE_NAMESPACE)
```

In the example here we pass the first two of those, as a 2-tuple. 

When you include a module the list of patterns comes from `urlpatterns`, and the application namespace comes from `app_name`. 

I've just used `app_name` here as any old string. It's usually, `'admin'`, or `'polls'` or `'rest_framework'`. 

In this example the instance namespace ends up being the same as the application namespace, because we didn't provide one. (This means that it'll be `None` if the application namespace is, which is just the global namespace we all start out with.) 

But we could do provide, say, `namespace='v1'` and such.


Grrr. I'm still not happy here.